src/tests/ilverify/TestDataLoader.cs

```diff
@@ -176,20 +176,37 @@ private static TheoryData<TestCase> GetTestMethodsFromDll(Func<string[], MethodD
                     var method = (EcmaMethod)testModule.GetMethod(methodHandle);
                     var methodName = method.Name;
 
-                    if (!String.IsNullOrEmpty(methodName) && methodName.Contains("_"))
+                    if (methodName?.Contains('_', StringComparison.Ordinal) == false)
```

Comment:
```suggestion
                    if (!methodName.Contains('_', StringComparison.Ordinal))
```
`EcmaMethod.Name` should not ever be null.